package.json

```diff
@@ -121,6 +121,7 @@
     "jest-watch-typeahead": "^0.4.2",
     "lerna": "^3.20.2",
     "lint-staged": "^10.0.8",
+    "mkdirp": "^1.0.4",
```

Comment:
Why do not use https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback with `recursive`?